configpm

```diff
@@ -576,14 +576,30 @@ $_ = <<'!END!';
 EOT
 #proper lexicographical order of the keys
 my %seen_var;
+my %v_define;
+my $defines = join " ", (Internals::V)[0,1];
+if (
+    $defines =~ /\b(SILENT_NO_TAINT_SUPPORT)\b/ ||
+    $defines =~ /\b(NO_TAINT_SUPPORT)\b/
+){
+    my $which = $1;
+    $v_define{taint_disabled} = ($which eq "SILENT_NO_TAINT_SUPPORT")
+                                ? "silent" : "define";
+    $v_define{taint_support}= "";
+}
```

Comment:
Win32 doesn't built miniperl with the same build options as the main perl binary/DLL, so if I build with -DNO_TAINT_SUPPORT:
```
> gmake CCHOME=c:\strawberry-perl-5.32.1.1-64bit-portable\c BUILDOPT=-DNO_TAINT_SUPPORT
...
>..\perl -I..\lib -V:taint_disabled
taint_disabled='';
...
>..\perl -I..\lib -V | ..\perl -ne "print if /TAINT/"
    ccflags =' -DWIN32 -DWIN64 -DNO_TAINT_SUPPORT'
    NO_TAINT_SUPPORT
>git status
On branch taint_disabled
Your branch is up to date with 'origin/yves/taint_disabled'.

nothing to commit, working tree clean
```

Reply:
Thanks for the feedback. This should be fixed in the latest push.